packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts

```diff
@@ -1369,6 +1369,25 @@ export = {
     test.done();
   },
 
+  'fromCodebuildImage'(test: Test) {
+    const stack = new cdk.Stack();
+    new codebuild.PipelineProject(stack, 'Project', {
+      environment: {
+        buildImage: codebuild.LinuxBuildImage.fromCodebuildImage('aws/codebuild/standard:4.0')
+      },
+    });
+
+    expect(stack).to(haveResourceLike('AWS::CodeBuild::Project', {
+      "Environment": {
+        "Type": "LINUX_CONTAINER",
+        "ComputeType": "BUILD_GENERAL1_SMALL",
```

Comment:
Feel free to remove these 2 lines (I think this test is only concerned with the `"Image"` part).